src/components/About/WarningBanner.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'lodash';
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Headline,
+  List,
+  MessageBanner,
+} from '@folio/stripes-components';
+
+import { isVersionCompatible } from '../../discoverServices';
+import css from './About.css';
+
+const WarningBanner = ({
+  interfaces,
+  modules,
+}) => {
+  const modulesArray = useMemo(() => _.flatten(_.values(modules)), [modules]);
+
+  const allInterfaces = useMemo(() => {
+    return modulesArray.reduce((prev, curr) => Object.assign(prev, curr.okapiInterfaces), {});
+  }, [modulesArray]);
+
+  const missingModules = useMemo(() => {
+    return Object.keys(allInterfaces).reduce((prev, curr) => {
+      if (!interfaces[curr]) {
+        return [...prev, `${curr} ${allInterfaces[curr]}`];
+      } else {
+        return prev;
+      }
+    }, []);
+  }, [allInterfaces, interfaces]);
```

Comment:
@axelhunn  I don't think that these changes will decrease code complexity. IMHO one reduce looks better than map after filter

Reply:
It's more self-explanatory for my taste.
Also, it could be faster - see https://www.measurethat.net/Benchmarks/Show/7249/0/filter-map-vs-reduce

Plus, you could DRY out the mapper for both `missingModules` and `incompatibleModules`